src/qibo/backends/_clifford_operations.py

```diff
@@ -454,11 +462,14 @@ def cast(x, dtype=None, copy=False):
     return np.array(x, dtype=dtype, copy=copy)
 
 
-def _clifford_pre_execution_reshape(state):
+def _clifford_pre_execution_reshape(state, pack=False):
```

Comment:
```suggestion
def _clifford_pre_execution_reshape(state, pack: bool = False):
```
Also maybe add a docstring since there are more arguments in the funciton